vspace/src/main/webapp/WEB-INF/views/exhibition/aboutPagePublic.html

```diff
@@ -0,0 +1,38 @@
+
+<html layout:decorate="~{layouts/main}">
+<head>
+<style>
+.main-contents {
+justify-content:center;
+margin-top:-20px;
+margin-left:100px;
+margin-right:100px;
+}
+
+
+</style>
+</head>
+
+<body >
+
+<div layout:fragment="content"  >
+
+<div id="Module_1" class="main-contents" >
+     <div style = "float:left">
+     <a href="javascript:history.back()" class="btn primary-btn btn-large" style="float: left; ">Back</a>
+     </div>       
+		
+	<div style="justify-content:center; float:left"> 
```

Comment:
why are the two divs float left? they are not meant to be next to each other. Also, give it a little bit more space between the heading and the button. This looks unnecessarily squished.